Makefile

```diff
@@ -1,80 +1,5 @@
+IMG ?= "crossplane/stack-minimal-aws"
```

Comment:
Nitpick: could we use `STACK_VERSION` to have the version of the stack, and `STACK_IMG` to have the `name:tag` combination? Similar to how it's being done in the minimal gcp stack now - https://github.com/crossplane/stack-minimal-gcp/blob/master/Makefile . That makes it idiomatic for the current stack tooling.

Reply:
Thanks for noticing, fixed!